### ISSUE TITLE
Pylance  complains about KeycloakConfiguration model 

### ISSUE THREAD

**Issue by JoseMMerida** [waza-ari/fastapi-keycloak-middleware#19]
The model has to include the default parameters as a kwarg

`keycloak_config = KeycloakConfiguration(
    url="https://sso.your-keycloak.com/auth/",
    realm="<Realm Name>",
    client_id="<Client ID>",
    client_secret="<Client Secret>",
)`

Complains about missing all the parameters which are optionals.

As an example:

`class KeycloakConfiguration(BaseModel):
  ...
  claims: list[str] = Field(
          default=[
              "sub",
              "name",
              "family_name",
              "given_name",
              "preferred_username",
              "email",
          ],
          title="Claims",
          description="The claims to add to the user object.",
    )
  ...`

Fixes the issue

**Comment by waza-ari** [comment]
Good morning,

Thank you for reporting this issue. `KeycloakConfiguration` is a Pydantic model, and I was under the impression that the first argument given to `Field` would specify the default value, as for example here:

```python
    reject_on_missing_claim: bool = Field(
        True,
        title="Reject on Missing Claim",
        description="Whether to reject the request if a claim is missing.",
    )
    authentication_scheme: str = Field(
        "Bearer",
        title="Authentication Scheme",
        description="The authentication scheme to use.",
    )
```

[According to the Pydantic docs](https://docs.pydantic.dev/2.3/usage/fields/#default-values) you're right though, they should be passed as `default=` kwarg. Not sure why I was under the impression that the first argument would be the default value and also not sure why the default values do work as they are. I'll investigate this a bit more, but won't find time today. Will look into it tomorrow. 

Do you happen to have some hint why the default values are currently applied correctly? I was looking into old Pydantic docs, but could not find references in the 1.x docs or the v2 migration guide either, so its nothing I have missed with Pydantic 2.

**Comment by JoseMMerida** [comment]
According to the Field source code it is just the first field of the Field, no extra magic there. TBH I think it is something that has to do with the linter implementations fo the type checking. The code is fully functional!

**Comment by waza-ari** [comment]
Thanks. I'll look into that on Sunday and cut a new release once changed.

**Comment by waza-ari** [comment]
Luckily for me I did not specify which "Sunday" - my apologies. It should be fixed with #36, I'll cut a new release later today.
